### PR TITLE
use install sdk binary script from sdk repository

### PR DESCRIPTION
**Pull request by smacker**
Part of https://github.com/src-d/lookout-sdk/issues/16

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,31 +10,35 @@ stages:
...
-
 jobs:
   include:
-    - name: 'Unit tests'
-      script:
-        - echo "TODO"
-
     - name: 'Integration test: analyzer server'
       install:
+        - docker run -d --name bblfshd --privileged -v $HOME/bblfshd:/var/lib/bblfshd -p "9432:9432" bblfsh/bblfshd:v2.9.2
```
Comment (carlosms) [inline .travis.yml]: Why is this needed now and not before?

Comment (smacker) [reply .travis.yml]: It was needed before. Look at any travis log before this commit. Analyzer was just crashing.

Comment (carlosms) [reply .travis.yml]: I just triggered a build on master and it's green https://travis-ci.org/src-d/lookout-sonarcheck-analyzer/builds/458384750, but yes the logs show the error "WantUAST isn't allowed". Should we open an issue to detect this failures?

Comment (smacker) [reply .travis.yml]: yes. And without uast the analyzer crashing. From you run:
```
grpc.service=pb.Data grpc.start_time="2018-11-22T12:28:33Z" source="grpchelper/logger.go:19" span.kind=server system=grpc
Exception calling application: <_Rendezvous of RPC that terminated with:
	status = StatusCode.UNKNOWN
	details = "Data server was started without bbflsh. WantUAST isn't allowed"
	debug_error_string = "{"created":"@1542889713.933854374","description":"Error received from peer","file":"src/core/lib/surface/call.cc","file_line":1083,"grpc_message":"Data server was started without bbflsh. WantUAST isn't allowed","grpc_status":2}"
>
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/grpc/_server.py", line 389, in _call_behavior
    return behavior(argument, context), True
  File "sonarcheck_analyzer.py", line 51, in NotifyReviewEvent
    for change in changes:
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/grpc/_channel.py", line 345, in __next__
    return self._next()
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/grpc/_channel.py", line 339, in _next
    raise self
grpc._channel._Rendezvous: <_Rendezvous of RPC that terminated with:
	status = StatusCode.UNKNOWN
	details = "Data server was started without bbflsh. WantUAST isn't allowed"
	debug_error_string = "{"created":"@1542889713.933854374","description":"Error received from peer","file":"src/core/lib/surface/call.cc","file_line":1083,"grpc_message":"Data server was started without bbflsh. WantUAST isn't allowed","grpc_status":2}"
>
time="2018-11-22T12:28:33.944333646Z" level=info msg="streaming client call finished" app=lookout-sdk duration=37.197256ms error="rpc error: code = Unknown desc = Exception calling application: <_Rendezvous of RPC that terminated with:
	status = StatusCode.UNKNOWN
	details = "Data server was started without bbflsh. WantUAST isn't allowed"
	debug_error_string = "{"created":"@1542889713.933854374","description":"Error received from peer","file":"src/core/lib/surface/call.cc","file_line":1083,"grpc_message":"Data server was started without bbflsh. WantUAST isn't allowed","grpc_status":2}"
>" grpc.code=Unknown grpc.method=NotifyReviewEvent grpc.service=pb.Analyzer grpc.start_time="2018-11-22T12:28:33Z" source="grpchelper/logger.go:19" span.kind=client system=grpc
```

with bblfsh it works successfully. We do need to improve checks. Maybe in a similar way to the one in lookout-sdk

Comment (carlosms) [reply .travis.yml]: Ok I created a new issue for it in #11.
